### DISCUSSION TITLE
Chiavetta ConBee II non trova lampadine

### DISCUSSION THREAD

**Discussion by federicobravin03** [indomus/forum#683] [HUB: Home Assistant]
Ciao a tutti, ho di recente acquistato questa chiavetta zigbee per collegare le mie diverse lampadine philips hue ad HA. Ho rieseguito più volte la ricerca dei dispositivi, e inoltre, ho provato a rifare la procedura di installazione, ma nulla è cambiato. Vi ringrazio in anticipo.   

Ciao.

**Comment by Moxser** [comment]
Ciao 
Hai resettato le Lampade della Philips prima di fare il paring, con app, telecomando o interruttore?

**Reply by federicobravin03** [reply]
No, potresti spiegarmi come fare? Perchè anche cercando su internet come fare il reset non ho trovato nulla.

**Reply by Moxser** [reply]
Ti scarichi l'app HUE sul cell. (sempre se non hai anche il telecomando)
Basta che spegni le lampadine e le riaccendi cosi le metti in paring, l'app le dovrebbe rilevare, da li le puoi resettare in modo che siano pronte per essere rilevate dalla chiavetta.

Comunque ci sono tantissime guide in rete su come resettarle, sia via app che tramite telecomando.

**Reply by federicobravin03** [reply]
Ho provato, ma non me le trova ancora.

**Reply by Moxser** [reply]
Ma cosa hai provato a fare?
Hai resettato una lampada alla volta? Per fare il pairing con la chiavetta? Io utilizzo una CC2530 ma penso che l'accoppiamento con le lampade zegbee sia il medesimo. Più di questo non ti so dire. 
Io generalmente quando devo inserire una nuova lampada o dispositivo zegbee faccio partire la rilevazione alla chiavetta spengo e accendo la lampada o metto in pairing il dispositivo che intendo accoppiare e me lo trova quasi istantaneamente.

**Comment by gsarcl366** [comment]
Io ho seguito questa strada ed ha funzionato. Vai sull'app Amazon Alexa, aggiungi dispositvo (fai tap su + in alto a destra), aggiungi dispositivo, seleziona Philips hue, successivamente fai tap sul punto interrogativo in alto a destra, fai scorrere la pagina sino alla voce "Resetta lampadina Philips Hue", seleziona e aggiungi in alto dove c'è scritto inserisci numero di serie, il numero di serie a 6 cifre vicino alla base della lampadina. Segui le indicazioni. Con me ha funzionato, contestualmente fai partire sull'addon la voce add new lights. Spero di esserti stato di aiuto. Giorgio Sarcletti
